lib/redirect.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react"
+import { useRouter } from "next/router"
+import languageDetector from "./languageDetector"
+
+export const useRedirect = (to) => {
+  const router = useRouter()
+  to = to || router.asPath
+
+  // language detection
+  useEffect(() => {
+    const detectedLng = languageDetector.detect()
```

Comment:
"detectedLang" could be more descriptive and clearer than "detectedLng"

Reply:
It was resolved